baseline: 0.1
gain: 1.0
k: 27.0
limit: 1.0
radius: 0.0318
v_max: 1.0
omega_max: 8.0
trim: 0
